### DISCUSSION TITLE
RKE2 Support Matrix link mentioned in https://docs.rke2.io/install/requirements does not work

### DISCUSSION THREAD

**Discussion by xplorenow** [rancher/rke2#6767] [General]
Link (https://www.suse.com/suse-rke2/support-matrix/all-supported-versions/rke2-v1-31/) which is supposed to be for RKE2 Support Matrix mentioned in https://docs.rke2.io/install/requirements does not work.

**Comment by brandond** [comment]
Tracking in https://github.com/rancher/rke2-docs/issues/255
